### PR TITLE
Fixes #37127 - Clean up old storybook dependencies

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: package-exclude.json [hunk 1]
```diff
@@ -15,16 +15,13 @@
         "babel-plugin-dynamic-import-node",
         "coveralls",
         "cross-env",
-        "highlight.js",
```
Comment (evgeni) [inline package-exclude.json:18]: those shouldn't be used in plugins and thus safe to remove now

File: package.json [hunk 2]
```diff
@@ -49,14 +49,11 @@
     "eslint": "^6.7.2",
     "eslint-plugin-spellcheck": "0.0.17",
     "graphql": "^15.5.0",
-    "highlight.js": "~9.14.0",
```
Comment (evgeni) [inline package.json:50]: This fixes https://github.com/advisories/GHSA-7wwv-vh3v-89cq, so @ehelms will be happy :)

**Review by ekohl — DISMISSED** [review]
If @MariaAga agrees I'd be happy to drop things

**Review by MariaAga — DISMISSED** [review]

**Comment by evgeni** [comment]
rebased after the node-sass drop

**Review by ekohl — APPROVED** [review]

**Comment by evgeni** [comment]
No idea why `test:integration` fails on GHA but not on Jenkins.
It's not caused by this PR, so I am going to ignore it for now.

https://github.com/theforeman/foreman/actions/workflows/foreman.yml?query=branch%3Adevelop

**Pull request merged**